var/spack/repos/builtin/packages/py-morpho-kit/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyMorphoKit(PythonPackage):
+    """Python library for reading / writing morphology files"""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/projects/nse/morpho-kit,dashboards/default"
+    url      = "https://bbpcode.epfl.ch/code/a/nse/morpho-kit"
+    git      = "ssh://bbpcode.epfl.ch/nse/morpho-kit"
+
+    version('develop', branch='master', submodules=True, clean=False)
+    version('0.1.0', tag='v0.1.0', submodules=True, clean=False)
+
+    depends_on('py-setuptools', type='build')
+
+    depends_on('morphio~mpi', type=('build', 'link'))
+
+    depends_on('cmake@3.2:', type='build')
+    depends_on('py-numpy', type='run')
```

Comment:
Question:  In our(hpc) stack , we typically have one package for c++ library and python package (e.g. in case of neuron, nmodl, steps etc.).

I wonder what's people opinion in general if single package can provide library as well as python bindings.

Reply:
> Question: In our(hpc) stack , we typically have one package for c++ library and python package (e.g. in case of neuron, nmodl, steps etc.).

Most of these are not primarily intended to be "native"-ish Python packages.

> I wonder what's people opinion in general if single package can provide library as well as python bindings.

I think the issue here is that NSE wants to have simple Python builds via `setup.py`, and provide pre-compiled wheels. This clashes with the traditional way of doing everything in CMake. I'm actually fine with having this split build.

Furthermore, I think that this simplifies the handling of MPI. For Python, `~mpi` is the default, for C++ `+mpi`.

Reply:
Ok